website_team/doc/index.rst

```diff
@@ -1,13 +1,23 @@
-=======
- Chess
-=======
+==============
+ Website Team
+==============
 
 Installation
 ============
 
-For Odoo 8
+* `Install <https://odoo-development.readthedocs.io/en/latest/odoo/usage/install-module.html>`__ this module in a usual way
 
 Usage
 =====
 
-**Note.** To use the module, you should install it and add necessary information about users you want to be represented
+**Usage**
+
+To add new information for user you should enter the website settings:
+    * 1 Click the "Website" button in the upper left corner of the window and click website button.
+    * 2 Click "Users" in the right menu.
+    * 3 Choose a user you want to edit and click the corresponding user line.
+    * 4 Click "Edit" button in the website header.
+Now you are able to add or edit it.
```

Comment:
Actually, I am not able to edit most information and I am not able to insert additional blocks